tools/astarte_export/lib/astarte/fetchdata/fetchdata.ex

```diff
@@ -1,19 +1,15 @@
 defmodule Astarte.Export.FetchData do
+  alias Astarte.Export.Utilities
   alias Astarte.Core.Device
   alias Astarte.Core.CQLUtils
   alias Astarte.Export.FetchData.Queries
 
-  def db_connection_identifier() do
-    with {:ok, conn_ref} <- Queries.get_connection() do
-      {:ok, conn_ref}
-    else
-      _ -> {:error, :connection_setup_failed}
-    end
-  end
-
   def fetch_device_data(conn, realm, opts) do
     with {:ok, result} <- Queries.stream_devices(conn, realm, opts),
-         [_device_data | _] = result_list <- Enum.to_list(result) do
+         [_device_data | _] = result_list <-
+           result
+           |> Enum.to_list()
+           |> Enum.map(&Utilities.map_string_to_atom/1) do
```

Comment:
`Enum.map` already transforms streams into enums, so we don't need the `to_list` instruction
also nit I personally find multi-line with matches hard to read


```suggestion
         [_device_data | _] = result_list <- Enum.map(result, &Utilities.map_string_to_atom/1) do
```